### DISCUSSION TITLE
configured debug type "pwa-extensionHost" is not supported

### DISCUSSION THREAD

**Discussion by yuanLeeMidori** [gitkraken/vscode-gitlens#1290] [Q&A] [answered]
Hi! I tried to run the repo with the instructions in CONTRIBUTING.md. When I press F5 to debug in VS code, I received the message box with **configured debug type "pwa-extensionHost" is not supported**. I didn't see familiar discussion in the open and closed issues and couldn't solve this problem.

Any help would be appreciated, thanks!
![image](https://user-images.githubusercontent.com/50813726/103056532-36e92b80-456b-11eb-8782-997d4c0bcf35.png)

**Comment by eamodio** [comment, marked as answer]
It sounds like your VS Code version might be old -- or maybe you have disabled the built-in _Javascript Debugger_?

**Reply by yuanLeeMidori** [reply]
Thank you... I updated the version to the latest one and this issue is gone. Sorry about that, I should have checked the version first. Thanks!

**Reply by eamodio** [reply]
No problem!
